docs_app/content/tutorial/4-what-is-an-operator.md

```diff
@@ -0,0 +1,71 @@
+
+## Collections
+
+The reason it is important to distinguish that an observable is a collection, is that **all collections have operations you can perform on them**. This actually jumps out of the realm of programming and into the rest of the universe. Imagine, for instance, you had a collection of apples. You could take that collection of apples, and "filter" out all of the bad apples, leaving you a collection of good apples. You could then "concatenate" them with a larger set of good apples you had "filtered" earlier. _Then_, you could take your good apples, and use a knife on them and "map" (or reduce) them into a collection of apple parts. _Then_ you could "filter" out the cores. You get the idea.
+
+Generally, you can always "map" a collection to a new collection, by virtue of the fact it has any values in it.
+
+You can do things like "filter" a collection as long as a new collection can be created of smaller size.
+
+You can do things like "concatenate" a collection with another collection _only_ if that collection has an "end". (Our apples have an end, in that there's presumably a finite number of apples).
+
+## Containers
+
+The idea of a [container](DICT) is simple. It holds things. Okay, easy right? But an interesting thing about containers is, some types of containers can hold "inner containers" of the same type. Boxes inside of boxes. Arrays inside of arrays. Observables inside of observables.
+
+If you have a container type, that can contains a collection of the same type of containers, that container can be _flattened_. That is to say, that you could systematically look in a container, pull out any containers you find, and dump their contents into a new container of the same type.
+
+Visually, in JavaScript that might look like this (shown with array literals):
+
+```ts
+[[1, 2, 3], [4, 5], [6]].flat() === [1, 2, 3, 4, 5, 6]
+```
+
+In terms of our apples above, that would be a box of boxes of apples, that you could then take out and dump into a new box of just apples.
+
+## Collections And Containers In JavaScript
+
+There are quite a few collections and containers already in existence in JavaScript. It's worth noting a few of them for comparisons here:
+
+### Arrays
+
+The most common collections of things we deal with in JavaScript are [Arrays](MDN_LINK_HERE). Arrays are collections that can be accessed synchronously, and randomly at indices. You can [map](MDN) an array into a new array of the same size. You can create an array of a smaller size (provided your current array isn't length of `0`), so it can be [filtered](MDN) to a new array. And arrays have a definitive "end" to them, which is represented in array literals with `]`, and at runtime by checking to see if your index is outside of the bounds provided by the `length`, therefor arrays can be [concatenated](MDN). Arrays can also be [flattened](MDN), as you can have an array of arrays.
```

Comment:
Consider flipping this sentence:

"As you can have an array of arrays, arrays can also be [flattened](MDN).